charts/pega/values-cdh.yaml

```diff
@@ -176,10 +176,10 @@ global:
           requests:
             storage: 5Gi
 
-    - name: "bix"
+    - name: "ADM"
```

Comment:
please spell out what ADM stands for and update the comments here as well->
> Create a background tier for BIX processing.  This tier uses
> the BIX node type and will not be exposed to the load balancer.

Reply:
+ standardize layer names to lowercase